recipes-connectivity: kuksa-client: update to v0.5.0

mlohvynenko opened the pull request.

mlohvynenko committed:
recipes-connectivity: kuksa-client: update to v0.5.0

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

al1img reviewed (approved):
`Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>`

mykola-kobets-epam commented:
`Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>`

mlohvynenko committed:
layers: pylibs: move kuksa-client to the aos pylibs layer

This patch moves the kuksa-client package from the main node's
rootfs to the aos-pylibs-layer.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>
